I think the NLS feature does not work.
This is a patch to fix the problem.

$OpenBSD: patch-configure_in,v 1.1 2002/11/21 14:56:31 brad Exp $
--- configure.in.orig	Mon Nov 18 05:59:23 2002
+++ configure.in	Wed Nov 20 19:35:23 2002
@@ -171,10 +171,12 @@ AC_ARG_ENABLE(nls,
   if test x$enableval = "xyes"; then
     AC_MSG_RESULT([yes])
     AC_DEFINE(NLS, 1, "Natural language support")
+    NLS="-DNLS"
   else
     AC_MSG_RESULT([no])
   fi,
   AC_MSG_RESULT([yes])
+  NLS="-DNLS"
   AC_DEFINE(NLS, 1, "Natural language support")
 )
 
@@ -184,6 +186,8 @@ AC_CHECK_PROGS(gencat_cmd, gencat)
 if test x$gencat_cmd = "x"; then
   NLS=""
 fi
+
+AC_SUBST(NLS)
 
 
 dnl Check for new timed pixmap cache
